### PR TITLE
Improve Check-in service integration guide

### PR DESCRIPTION
**Pull request by NicolasLiampotis**
This pull request aims to improve the integration documentation for EGI Check-in. The changes include providing additional clarity regarding the purpose and differences of the development environment compared to the demo and production environments. 

# Summary

A note has been added to highlight that the development environment is intended for testing the latest features and may differ from demo/prod in terms of stability and functionality. These changes aim to improve the understanding of the integration process and help users make informed decisions about the integration environment when testing their Service Providers.

### CODE REVIEW

**Review by vardizzo-lab — APPROVED** [review]
ok

**Comment by enolfc** [comment]
@NicolasLiampotis can you fix the reported broken link so we can merge?

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -1535,118 +1539,6 @@ $sessionName = "simple-oidc-client-php";
...
-
-- If you have obtained Refresh Tokens using the oidc-agent, please use the
-  following command:
-
-  ```shell
-  $ oidc-gen --pub --issuer <ISSUER> --scope ...
-  ```
-
-  {{% alert title="Note" color="info" %}} You can find the `ISSUER` in the
-  [Endpoints](#endpoints) table.{{% /alert %}}
```
Comment (enolfc) [inline content/en/providers/check-in/sp/_index.md:1542]:
```suggestion
```

**Review by vardizzo-lab — COMMENTED** [review]
OK.

**Pull request merged**